tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LraResource.java

```diff
@@ -135,10 +140,13 @@ public Response leaveWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
     @Path("/complete")
     @Produces(MediaType.APPLICATION_JSON)
     @Complete
-    public Response completeWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId, String userData) {
+    public Response completeWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId,
+                                 @HeaderParam(LRA_HTTP_RECOVERY_HEADER) URI recoveryId,
+                                 String userData) {
         lraMetricService.incrementMetric(LRAMetricType.Completed, lraId, LraResource.class.getName());
 
-        assertHeaderPresent(lraId); // the TCK expects the coordinator to invoke @Completed methods
+        assertHeaderPresent(lraId, LRA_HTTP_CONTEXT_HEADER); // the TCK expects the coordinator to invoke @Completed methods
+        assertHeaderPresent(recoveryId, LRA_HTTP_RECOVERY_HEADER); // the TCK expects the coordinator to invoke @Completed methods
```

Comment:
same as above, we should not mention coordinator explicitly, and typo "`@Complete` methods"